runtime/oti/j9nonbuilder.h

```diff
@@ -5480,6 +5480,7 @@ typedef struct J9VMThread {
 	UDATA discontiguousIndexableHeaderSize;
 #if defined(J9VM_ENV_DATA64)
 	UDATA isIndexableDataAddrPresent;
+	BOOLEAN isVirtualLargeObjectHeapEnabled;
```

Comment:
we could have this flag under the compile time flag, which would mean that appropriate access macros that we later introduce would be under the same flag.

 @dmitripivkine, opinion, do we want it?

Reply:
after some discussion, seems like call sites (array access macros) will be a bit more complex and harder to read, with little benefit (one less field in VM Thread, if build flag is disabled, which  would effectively be interesting only for 32bit platforms)
so let's leave it as-is, but we can always add it later if we want

Reply:
With the addition of this field, should line 5517 change?
```c
#if 0 && !defined(J9VM_ENV_DATA64) /* Change to 0 or 1 based on number of fields above */
```

FYI @gacholio

Reply:
We should add an assertion into the build process to detect this,

Reply:
Was the alignment resolved?

Reply:
Seems like that line should be updated only for non 64 bit platforms, and we added a field for 64 bit only.

Reply:
Yes, I see that now. I still think @gacholio's suggestion to have a build-time check is a good one.